src/referrals/referrals.controller.ts

```
@@ -129,6 +129,22 @@ export class ReferralsController {
     );
   }
 
+  @UseFilters(new AllExceptionsFilter(APIID.REFERRAL_SUMMARY))
+  @Post('report/summary')
```

ESLint

[error] Replace `'report/summary'` with `"report/summary"`

(prettier/prettier)